wa/workloads/gmail/uiauto/app/src/main/java/com/arm/wa/uiauto/gmail/UiAutomation.java

```diff
@@ -189,58 +229,28 @@ public void attachImage() throws Exception {
         if (rootMenu.exists()){
             rootMenu.click();
         }
-        // Check for Photos
```

Comment:
Not all devices have the "Images" entry in the sidebar (testing on nexus5) and therefore still rely on GooglePhotos. I would suggest leaving the use of google photos as a fall back option.